types/mongodb/index.d.ts

```diff
@@ -1217,10 +1218,10 @@ export interface Collection<TSchema extends { [key: string]: any } = DefaultSche
         callback: MongoCallback<UpdateWriteOpResult>,
     ): void;
     /** http://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#watch */
-    watch(
+    watch<TSchema extends object = {_id: ObjectId}>(
```

Comment:
Is there a reason why this isn't `<T = TSchema>`, the same as `aggregate`?

Also, probably best to not name this `TSchema` since that type parameter is already present on the entire interface.